Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```
@@ -191,6 +191,14 @@ script:
       name: additional_fields
       required: false
       secret: false
+    - default: false
+      defaultValue: ;
+      description: The delimiter character to use to separate any fields list argument
```

WDYT about this? if ok please tend to in all of the occurrences.
```suggestion
      description: The delimiter character to use to separate all of the fields provided in the custom_fields argument. Default is ";".
```

---

The problem is, not only the 'custom_fields' argument uses the split method. There are some other arguments that are separated with semicolons. So I think that if the customer has changed this argument it should apply to all relevant arguments.

---

> WDYT about this? if ok please tend to in all of the occurrences.
> 
> ```suggestion
> 
>       description: The delimiter character to use to separate all of the fields provided in the custom_fields argument. Default is ";".
> 
> ```

This sounds good. Let's apply this.

---

> The problem is, not only the 'custom_fields' argument uses the split method. There are some other arguments that are separated with semicolons. So I think that if the customer has changed this argument it should apply to all relevant arguments.

I'm inclined to agree here. If anything we could add the delimiter argument to all pertinent commands with the default set to `;`, but give the customer the option to override it.